addons/im_livechat/models/discuss_channel.py

```diff
@@ -60,8 +60,10 @@ def _channel_info(self):
                 'name': channel.country_id.name,
             } if channel.country_id else False
             if channel.livechat_operator_id:
-                display_name = channel.livechat_operator_id.user_livechat_username or channel.livechat_operator_id.display_name
-                channel_infos_dict[channel.id]['operator_pid'] = (channel.livechat_operator_id.id, display_name.replace(',', ''))
+                fields_list = {'id': True, 'name': True, 'write_date': True}
+                if channel.livechat_operator_id.user_livechat_username:
+                    fields_list = {'id': True, 'user_livechat_username': True, 'write_date': True}
```

Comment:
It's strange to repeat the other keys, only name should be concerned by the condition.